lib/appraisal/group.rb

```diff
@@ -1,42 +1,19 @@
-require 'appraisal/dependency_list'
+require 'appraisal/bundler_dsl'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.